python/sparkdl/param/shared_params.py

```diff
@@ -232,7 +232,7 @@ class HasTFHParams(Params):
                       key-value object, storing parameters to be used to define the final
                       TensorFlow graph for the Transformer.
 
-                      Currently accepted values are:
+                      Currently used values are:
```

Comment:
you can make the change in a later PR, but let's add that these are optional (they are, right?)